app/models/role.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Role < ApplicationRecord
-  has_many :users, through: :users_roles
+  has_and_belongs_to_many :users, join_table: :users_roles
```

Comment:
Umm…these may not be the same macros…let's be cautious.  But there's no `UserRole` model (nor `RoleUser`) so; I guess we'll be alright with this.

Reply:
They are NOT the same... that's why I had to revert it back to this and exclude it in rubocop yml.